tests/testutils.py

```diff
@@ -57,9 +57,7 @@ def subprocess_run(*args: str, **kwargs: Any) -> subprocess.CompletedProcess:
     Helper method to run a subprocess. Asserts for success.
     """
     compat_kwargs = {"text" if PY37 else "universal_newlines": True}
-    result = subprocess.run(
-        args, **compat_kwargs, stdout=subprocess.PIPE, stderr=subprocess.PIPE, **kwargs
-    )
+    result = subprocess.run(args, **compat_kwargs, capture_output=True, **kwargs)
```

Comment:
should be able to drop `compat_kwargs`:
```suggestion
    result = subprocess.run(args, text=True, capture_output=True, **kwargs)
```

Reply:
Good catch